packages/server/src/index.ts

```diff
@@ -599,10 +605,76 @@ export class App {
             if (sessionId) deleteOptions.sessionId = sessionId
             if (chatType) deleteOptions.chatType = chatType
 
+            // remove all related feedback records
+            const feedbackDeleteOptions: FindOptionsWhere<ChatMessageFeedback> = { chatId }
+            await this.AppDataSource.getRepository(ChatMessageFeedback).delete(feedbackDeleteOptions)
+
             const results = await this.AppDataSource.getRepository(ChatMessage).delete(deleteOptions)
             return res.json(results)
         })
 
+        // ----------------------------------------
```

Comment:
@ocflowiseai also new endpoints that needs to be updated once its merged - https://github.com/FlowiseAI/Flowise/pull/1806